tests/test_build_linkcheck.py

```diff
@@ -728,3 +733,44 @@ def test_linkcheck_exclude_documents(app):
             'info': 'br0ken_link matched br[0-9]ken_link from linkcheck_exclude_documents',
         },
     ]
+
+
+urls = [
```

Comment:
I would advise not to use a global declaration here and simply write

```python
@pytest.mark.parametrize("test_url", [
    "...."
])
```

Also, maybe rename "123" to "anchor" so that we know that it is an anchor.